notebooks/1_LinearMethods.ipynb

```diff
@@ -227,16 +229,9 @@
     "\\mathbf{T} = \\mathbf{X}\\hat{\\mathbf{U}}_C,\n",
     "\\end{equation}\n",
     "\n",
-    "PCA projections give a simplified representation of the data that may (or may not!) correlate with the property of interest, but should include most of the variance in the feature space."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "print(f\"Our low-dimensional space contains {n_PC} components.\")"
+    "PCA projections give a simplified representation of the data that may (or may not!) correlate with the property of interest, but should include most of the variance in the feature space.\n",
+    "\n",
+    "Our low-dimensional space contains **{{n_PC}}** components."
```

Comment:
Do we want to note anywhere that in order for this to render properly you need the "Python Markdown" NB extension enabled? Mine wasn't by default.

Reply:
I added a line in the Forward about dependencies and python markdown.